insonmnia/worker/server.go

```diff
@@ -949,6 +970,11 @@ func (m *Worker) setupRunningContainers() error {
 				continue
 			}
 
+			info.Cinfo.ContainerInfo.PublicKey, _ = ssh.ParsePublicKey(info.Cinfo.PublicKey)
+
+			info.Description.Description.networks = info.Description.Networks
```

Comment:
looks weird

Reply:
it does, but yet in the existing code https://github.com/sonm-io/core/blob/master/insonmnia/worker/server.go#L601 ```ssh.ParsePublicKey()``` is not checked for returned error, as you may see. In the event when taskinfo has no ssh key that would be the case and containerInfo.PublicKey whill be ```nil```. So when we restore the task the same logic has to be applied: if no key is in the storage, containerInfo.PublicKey is also ```nil```. This logic definitely has to be fixed, but I don't that this PR is the place to do it.